src/components/Annotation/Annotation.js

```diff
@@ -0,0 +1,119 @@
+import CommentInlineError from '../../../third-party/components/CommentInlineError';
+import UserLink from '../../../third-party/components/UserLink';
+import messages from './messages';
+
+import './Annotation.scss';
+
+const ONE_HOUR_MS = 3600000; // 60 * 60 * 1000
+
+type Props = {
+    id: string,
+    message: string,
+    permissions: AnnotationPermissions,
+    createdAt: string,
+    createdBy?: User,
+    modifiedAt?: string,
+    onDelete?: Function,
+    isPending?: boolean,
+    error?: ActionItemError
+};
+
+type State = {
+    isFocused?: boolean
+};
+
+class Annotation extends React.Component<Props, State> {
+    state = {
+        isFocused: false
+    };
+
+    onKeyDown = (event: SyntheticKeyboardEvent<>): void => {
+        const { nativeEvent } = event;
+        nativeEvent.stopImmediatePropagation();
+    };
+
+    handleCommentFocus = (): void => {
+        this.setState({ isFocused: true });
+    };
+
+    handleCommentBlur = (): void => {
+        this.setState({ isFocused: false });
+    };
+
+    render() {
+        const {
+            id,
+            createdAt,
+            createdBy,
+            permissions,
+            message,
+            isPending = false,
+            error,
+            onDelete = noop
+        } = this.props;
+        const { isFocused } = this.state;
+        const canDelete = getProp(permissions, 'can_delete', false);
+        const createdAtTimestamp = new Date(createdAt).getTime();
+
+        return (
+            <div className='ba-annotation-container'>
+                <div
+                    className={classNames('ba-annotation', {
+                        'ba-is-pending': isPending || error,
+                        'ba-is-focused': isFocused
+                    })}
+                    onBlur={this.handleCommentBlur}
+                    onFocus={this.handleCommentFocus}
+                >
+                    <Avatar className='ba-annotation-avatar' {...createdBy} />
+                    <div className='ba-annotation-content'>
+                        <div className='ba-annotation-headline'>
+                            {createdBy && createdBy.name ? (
+                                <UserLink className='ba-annotation-user-name' id={createdBy.id} name={createdBy.name} />
+                            ) : (
+                                <FormattedMessage className='ba-annotation-user-name' {...messages.anonymousUserName} />
+                            )}
+                            <Tooltip
+                                text={
+                                    <FormattedMessage
+                                        {...messages.annotationPostedFullDateTime}
+                                        values={{ time: createdAtTimestamp }}
+                                    />
+                                }
+                            >
+                                <small className='ba-annotation-created-at'>
```

Comment:
Should this instead be a `<time>` tag? Also, this should only be wrapped in a `<small>` tag if we're trying to de-emphasize the time. Otherwise styling should be done via CSS.